.travis.yml

```diff
@@ -5,21 +5,18 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
-  
+
 matrix:
   allow_failures:
     - php: hhvm
+    - php: 7.0
 
-script:
-  - phpunit -c phpunit.xml.dist
-  - ./run-functional-tests
-  
 before_install:
   - sudo apt-get update && sudo apt-get --reinstall install -qq language-pack-en language-pack-fr && sudo locale-gen fr_FR
 
 before_script:
-  - composer self-update || true
```

Comment:
Travis' default composer is most often out of date. This is extremely valid to make sure we're testing against a current release.